client/beefy/src/worker.rs

```diff
@@ -400,6 +409,11 @@ where
 		));
 
 		loop {
+			if self.sync_oracle.is_major_syncing() {
+				debug!(target: "beefy", "Skipping initialization due to sync.");
```

Comment:
This is not necessarily initialization, it is the main loop of the worker. I would rephrase to smth like:

```suggestion
				debug!(target: "beefy", "Waiting for major sync to complete.");
```